packages/Treasury/src/Services/TreasuryPositionService.php

```
@@ -0,0 +1,289 @@
+            (new DateTimeImmutable())->format('Y-m-d')
+        );
+
+        $paymentPeriod = $this->payableProvider->getAveragePaymentPeriod($tenantId);
+        if ($paymentPeriod <= 0) {
+            return 0.0;
+        }
+
+        return $totalPayables / $paymentPeriod;
+    }
+
+    private function getDefaultCurrency(string $tenantId): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$tenantId'. (undefined)

(UnusedFormalParameter)